modules/meeting/app/components/meetings/sidebar/details_component.rb

```diff
@@ -128,14 +128,23 @@ def duration_partial
       duration = Duration.new(seconds: @meeting.duration * 3600)
 
       meeting_attribute_row(:stopwatch) do
-        if duration.hours > 0
-          render(Primer::Beta::Text.new) do
-            "#{duration.hours} h #{duration.minutes} min"
-          end
-        else
-          render(Primer::Beta::Text.new) do
-            "#{duration.minutes} min"
+        flex_layout do |flex|
+          flex.with_column do
+            if duration.hours > 0
+              render(Primer::Beta::Text.new) do
+                "#{duration.hours} h #{duration.minutes} min"
+              end
+            else
+              render(Primer::Beta::Text.new) do
+                "#{duration.minutes} min"
+              end
```

Comment:
I am aware that this existed before but it would still make sense to extract the translation into the locale yml and then rely on e.g. I18n.l for translations of the Duration object. This could then be reused for the duration displayed for the AgendaItem.

In general, it would simplify the code if the ruby `Duration` or `ActiveSupport::Duration` where to be used as this would stop the calculations being done (`* 60`, ` * 3600`).